README.md

```diff
@@ -60,7 +60,6 @@ Put game assembly `Assembly-CSharp.dll` to `lib/` and use `dotnet build`
 ## Known issues
 
  * This mod may crash the game if you have malformed option data installed with missing music jackets.
- * This mod relies on specific scene sequences to synchronize state, and scene-skipping mods may cause it to malfunction.
```

Comment:
Please don't remove this warning since we still relies on specific hooks to work correctly and we never know what the user might have done to their games. This warning will be useful for troubleshooting.